tests/unit/test_util.h

```diff
@@ -60,6 +63,15 @@ extern "C" {
     } \
   } while (0)
 
+//#define CompileTimeAssert(COND, MSG) typedef char __C_ASSERT__[(COND)?1:-1]
+   
+// I would use the above definition for CompileTimeAssert, but I am getting the following errors
+// when I run 'make -f examples/Makefile-posix distcheck':
+//
+//      error: typedef �__C_ASSERT__� locally defined but not used [-Werror=unused-local-typedefs]
+//
+#define CompileTimeAssert(COND, MSG)
```

Comment:
I'm OK with punting on getting this working in other platforms for now. Let's file a bug report after this gets merged in so that we don't forget about it.